### PR TITLE
Update node layout per Elastic recommendations

### PR DESCRIPTION
**Pull request by rocketraman**
Update node layout per Elastic recommendations. Solves issue #190 .

### CODE REVIEW

**Comment by pires** [comment]
Actually, the instructions are incomplete, e.g. you renamed `es-client.yaml` to `es-ingestion.yaml`. But I'll take care of it :)

**Comment by rocketraman** [comment]
@pires Sorry about that, I've updated the second README commit. Should be good now!

File: es-data.yaml [hunk 1]
```diff
@@ -52,13 +52,21 @@ spec:
...
         - containerPort: 9300
           name: transport
         livenessProbe:
           tcpSocket:
             port: transport
           initialDelaySeconds: 20
           periodSeconds: 10
+        readinessProbe:
+          httpGet:
+            path: /_cluster/health
```
Comment (pires) [inline es-data.yaml]: You didn't even try this because `HTTP_ENABLE` is still `false`.

File: stateful/es-data-stateful.yaml [hunk 2]
```diff
@@ -53,13 +53,21 @@ spec:
...
         - containerPort: 9300
           name: transport
         livenessProbe:
           tcpSocket:
             port: transport
           initialDelaySeconds: 20
           periodSeconds: 10
+        readinessProbe:
+          httpGet:
+            path: /_cluster/health
```
Comment (pires) [inline stateful/es-data-stateful.yaml]: Again, it's clear you didn't even try this because HTTP_ENABLE is still false.

Comment (rocketraman) [reply stateful/es-data-stateful.yaml]: Thanks, yeah, I was diffing from the local changes I had made in another repo, and I missed this one. Sorry about that.

**Comment by rocketraman** [comment]
@pires BTW, the readme changes in 7c440dc don't seem to have survived your merging process. The README on master still refers to client nodes.

**Comment by pires** [comment]
Correct! Fixed, thanks.

**Pull request closed without merging**